### PR TITLE
Lambda zdc

### PR DESCRIPTION
**Pull request by sebouh137**
### Briefly, what does this PR introduce?
adds a new benchmark, detecting lambda baryons in the ZDC through their decay into nπ0 -->nγγ.   

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [X] Other:  new benchmark

### Please check if this PR fulfills the following:
- [X] Tests for the changes have been added
- [X] Documentation has been added / updated
- [X] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
no
### Does this PR change default behavior?
Yes.  Adds new benchmark

### CODE REVIEW

File: benchmarks/lambda/Snakefile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+rule lambda_recon:
+        input:
+                SIM_FILE="{DETECTOR_CONFIG}_sim_lambda_dec_{P}GeV.edm4hep.root"
+        params:
+                NEVENTS_REC=3000,
+        output:
+                REC_FILE="{DETECTOR_CONFIG}_rec_lambda_dec_{P}GeV.edm4hep.root"
+        shell:
+                """
+eicrecon -Pplugins=ZDC {input.SIM_FILE} -Ppodio:output_file={output.REC_FILE} -Pdd4hep:xml_files=$DETECTOR_PATH/{wildcards.DETECTOR_CONFIG}.xml -Ppodio:output_include_collections=MCParticles,HcalFarForwardZDCClusters,HcalFarForwardZDCRecHits,HcalFarForwardZDCSubcellHits  -Pjana:nevents={params.NEVENTS_REC}
```
Comment (veprbl) [inline benchmarks/lambda/Snakefile:41]: `-Pplugins=ZDC` is broken in recent JANA2 https://chat.epic-eic.org/main/pl/mmyspc5uhfrp7khu1tz9p8kr8o
Let's just not use it
```suggestion
eicrecon {input.SIM_FILE} -Ppodio:output_file={output.REC_FILE} -Pdd4hep:xml_files=$DETECTOR_PATH/{wildcards.DETECTOR_CONFIG}.xml -Ppodio:output_include_collections=MCParticles,HcalFarForwardZDCClusters,HcalFarForwardZDCRecHits,HcalFarForwardZDCSubcellHits  -Pjana:nevents={params.NEVENTS_REC}
```

**Comment by sebouh137** [comment]
@wdconinc ,
I am trying to figure out why a benchmark that I have written is failing, and it seems as though the second stage (creating plots) is unable to find the files created in the first stage (root files created in EICrecon).  Could you look over my Snakefile and config.yml and tell me why this is happening?  Are these files simply not being persisted from one stage to the next?  I don't have much experience in making benchmarks, so I am not sure what's going wrong.  
Thanks,
Sebouh

**Comment by sebouh137** [comment]
> @wdconinc , I am trying to figure out why a benchmark that I have written is failing, and it seems as though the second stage (creating plots) is unable to find the files created in the first stage (root files created in EICrecon). Could you look over my Snakefile and config.yml and tell me why this is happening? Are these files simply not being persisted from one stage to the next? I don't have much experience in making benchmarks, so I am not sure what's going wrong. Thanks, Sebouh

Update:  talked to @c-dilks .   He explained, the output files need to go to the results folder or better a subfolder thereof.  This has been resolved.

**Comment by veprbl** [comment]
This looks good. However we need to resolve one operational issue. The root and hepmc files can not go to `results/` as that results in large files being uploaded as artifacts. I don't know the details, but I'm told this is not what eicweb prefers currently.

**Comment by veprbl** [comment]
See #21

**Comment by wdconinc** [comment]
It's an efficiency thing and gets your jobs to run faster. If you don't have to upload a large artifact and the next job doesn't have to download it, then you probably gain a few seconds. And it saves bandwidth on eicweb.

**Comment by veprbl** [comment]
Right. But I think there is also a storage space issue. I remember Sylvester saying that getting a local S3-backed storage for artifacts is considered to allow extending their use.

**Comment by wdconinc** [comment]
What size is this artifact? We're only storing the artifacts for 72 hours... And we're storing GBs of container sifs... But yeah, there's some aspect of storage too.

**Pull request merged**